src/protocols/interaction_model/StatusCodeList.h

```diff
@@ -74,5 +74,7 @@ CHIP_IM_STATUS_CODE(TimedRequestMismatch  , TIMED_REQUEST_MISMATCH  , 0xc9)
 CHIP_IM_STATUS_CODE(FailsafeRequired      , FAILSAFE_REQUIRED       , 0xca)
 CHIP_IM_STATUS_CODE(InvalidInState        , INVALID_IN_STATE        , 0xcb)
 CHIP_IM_STATUS_CODE(NoCommandResponse     , NO_COMMAND_RESPONSE     , 0xcc)
+CHIP_IM_STATUS_CODE(SuppressReport        , SUPPRESS_REPORT         , 0xcd)
+CHIP_IM_STATUS_CODE(ForceReport           , FORCE_REPORT            , 0xce)
```

Comment:
This is the list of spec-defined status codes.  We can't just randomly add things to it....

In particular, 0xCD and 0xCE are in fact defined in the 1.4 spec and just haven't been added here yet.

Reply:
Noted.  Seems like this would be useful to other clusters having Q-quality attributes and not using the AttibuteAccessInterface?